packages/maker/wix/src/MakerWix.ts

```diff
@@ -22,13 +22,15 @@ export default class MakerWix extends MakerBase<MakerWixConfig> {
     const outPath = path.resolve(makeDir, `wix/${targetArch}`);
     await this.ensureDirectory(outPath);
 
-    let { version } = packageJSON;
-    if (version.includes('-')) {
+    const { version } = packageJSON;
+    if (version.includes('-') || version.includes('+')) {
```

Comment:
Needs `import semver from 'semver'` above but:

```suggestion
    const { version } = packageJSON;
    const parsed = semver.prerelease(version);
    if (Array.isArray(parsed) && parsed.length > 0) {
```

Reply:
I'm fine with using `semver` package to check the version components (I thought about it but didn't want to add it to have one more dependency just to show one log message).

But I intentionally added a new check for `+` because prerelease is not the only part of semver that will be changed.

Reply:
Ah, thanks! Didn't realize there was that distinction between `+` and `-` so I had to re-read the spec.

Since we're already including `semver` in many places across the monorepo (including `@electron-forge/core`, I think adding the dep to `maker-wix` is kind of "free" in a sense!